pkg/limayaml/defaults.go

```diff
@@ -622,6 +626,16 @@ func IsAccelOS() bool {
 	return false
 }
 
+func HasHostCPU() bool {
+	switch runtime.GOOS {
+	case "darwin", "linux":
+		return true
+	case "netbsd", "windows":
+		return false
```

Comment:
Why not let this be handled by the default?

Reply:
It was an effort at documentation, possibly not working

Reply:
OS without any specific `-accel` (beyond "tcg"), will never call this

Reply:
> It was an effort at documentation, possibly not very successful :-)

If it is documentation, then I'm not getting it. :)

It feels like something you will only understand if you already know it, so it doesn't make sense to have a special case for it, unless you also add a comment to explain why these are treated specially.

Reply:
The only idea was that we should have the same OS listed in all three places.

```go
                case "darwin":
                        return "hvf"
                case "linux":
                        return "kvm"
                case "netbsd":
                        return "nvmm" // untested
                case "windows":
                        return "whpx" // untested
```

But it wasn't obvious from the context, and it was actually tested.

Reply:
Added comment